guide/_sidebar.md

```diff
@@ -36,6 +36,7 @@
 	* **Database**
 	* [Storing data with Sequelize](/sequelize/)
 	* [Making a currency system](/sequelize/currency)
+	* [Building a per-guild prefix system](/keyv/)
```

Comment:
I think the sidebar link should be labeled differently, because if a user reads the first 2 pages about Sequelize and then clicks on this one, they'd assume it continues using Sequelize. And most of the page is more about using Keyv than just building a per-guild prefix system.

```suggestion
	* [Working with Keyv](/keyv/)
```